R/viz.R

```diff
@@ -207,23 +210,21 @@ prettyRainfallPlot = function(this_sample_id,
       )
     )
   }
-
+  
   # if user is subsetting by chromosome or zooming in to a specific region, are there any SSM left to plot?
   if (nrow(rainfall_points) == 0) {
     stop("After subsetting to a regions you requested to plot, there are no SSM to display.")
   }
-
+  
   # label SVs if user wants to overlap this data
   if (!missing(chromosome) & label_sv) {
     sv_chromosome = chromosome
   } else if (!missing(zoom_in_region) & label_sv) {
     sv_chromosome = zoom_in_region$chromosome
   } else if (label_sv) {
-    stop(
-      "Labeling SV is only supported when a particular chromosome or zoomed region is plotted."
-    )
+    sv_chromosome = 1:22
```

Comment:
I trust this works for the chr-prefixed too?

Reply:
good catch, should probably change to 
`sv_chromosome = standardize_chr_prefix(incoming_vector = c(1:22,"X","Y"), projection = projection)`